Formula/ahoy.rb

```diff
@@ -0,0 +1,35 @@
+class Ahoy < Formula
+  desc "Creates self documenting cli programs from commands in yaml files"
+  homepage "http://www.ahoycli.com/"
+  url "https://github.com/ahoy-cli/ahoy/archive/2.0.0.tar.gz"
+  sha256 "cc3e426083bf7b7309e484fa69ed53b33c9b00adf9be879cbe74c19bdaef027c"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/ahoy-cli/ahoy"
+    bin_path.install Dir["*"]
+    cd bin_path do
+      system "go", "build", "-o", bin/"ahoy", "-ldflags", "-X main.version=#{version}", "."
+    end
+  end
+
+  def caveats; <<~EOPLIST
```

Comment:
Use `EOS` (end of string) instead of `EOPLIST`.

Reply:
done